decidim-admin/app/controllers/decidim/admin/taxonomies_controller.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Admin
+    class TaxonomiesController < Decidim::Admin::ApplicationController
+      include Decidim::Admin::Filterable
+
+      layout "decidim/admin/settings"
+
+      helper_method :taxonomies, :parent_options, :taxonomy
+
+      def index
+        @query = base_query.ransack(params[:q])
+        @taxonomies = @query.result
```

Comment:
also this mixes with the `taxonomies` method, maybe we can compact this behavior